gobblin-runtime/src/main/java/org/apache/gobblin/runtime/GobblinMultiTaskAttempt.java

```diff
@@ -201,9 +202,19 @@ public void shutdownTasks()
       task.shutdown();
     }
 
-    for (Task task: this.tasks) {
+    for (Task task : this.tasks) {
       task.awaitShutdown(1000);
     }
+
+    for (Task task : this.tasks) {
+      if (task.getTaskFuture().cancel(true)) {
+        log.info("Task cancelled. Countdown and shutdown latches will be decreased here.");
```

Comment:
Can you please mention the task name which was cancelled here

Reply:
done.